index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Trump and Putin's buthole</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+<main>
+    <div class="topSection">
+        <div class="title">This is a title</div>
+    </div>
+    <section class="gameSection">
+
+        <div class="gameScreen">
+            <div class="Height30percent"></div>
+            <div class="Height30percent">
+                <div class="character">This is a character</div>
+                <div class="finishLine">This is the finish line</div>
+            </div>
+            <div class="Height30percent">
+                <div class="pumpkin">This is a talking pumpkin</div>
+                <div class="pumpkin">This is what the pumpkin is saying init</div>
+            </div>
+        </div>
+        <div type="button"> Start Game  </div>
```

Comment:
in the html this is not registering as a button, if this is not intentional you could make it into a <button> instead

Reply:
BUttons hold predefined styles which I don't want.